### DISCUSSION TITLE
Generating type safe usage of i18n keys

### DISCUSSION THREAD

**Discussion by osdiab** [sergiodxa/remix-i18next#96] [Show and tell]
Hi all, inspired by @Xiphe 's work on https://github.com/Xiphe/remix-polyglot , i put together a script that allows for a bit more typesafe use of this library. it doesn't yet include actually typing the variable interpolations, but it could :) anyway hope this helps someone out there.

https://gist.github.com/osdiab/23f197ae82621cadc07ee8bd57e5101e

**Comment by Xiphe** [comment]
Sweet! Thanks for sharing!
